src/components/ratingsReviews/RatingsPanel/OverallRating.tsx

```diff
@@ -29,9 +30,12 @@ const OverallRating: FC<OverallRatingProps> = ({ ratings }) => {
     }
   }
 
+  console.log()
+
   return (
     <div className="overall-rating">
-      { averageRating(ratings) }
+      <div>{ averageRating(ratings) }</div>
+      <div className="overall-stars"> < Rating readonly={true} initialValue={ Number(averageRating(ratings)) } size={ 18 } fillColor="#525252" emptyColor="#00000040" allowFraction={ true }/> </div>
```

Comment:
Bit of a nitpick but this would be slightly more readable if the Rating component started on its own line.